setup.py

```diff
@@ -187,6 +187,10 @@ def build_extension(self, ext: CMakeExtension):
 suffix = suffix[0].upper() + suffix[1:]
 
 pennylane_plugins = [device_name + " = pennylane_lightning." + backend + ":Lightning" + suffix]
+if suffix == "Qubit":
```

Comment:
I can't help but think it's funny that in the end we only needed these 4 lines in `setup.py`. Except for the changes that @mlxd requested, for me this is good to go